docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,38 @@
+# Default overrides for running local development.
```

Comment:
Move this to `docker/debug/docker-compose.yml` for consistency?

Reply:
I think having `docker/<service>` makes the most sense. `debug` isn't a service. Plus, this `yml` file is rather important.

Reply:
Since you have `development.md` maybe it makes sense to group that and `docker-compose.dev.yml` into a dev directory and keep the docker root cleaner?

Reply:
I intentionally moved the docs to docs/. imo consolidated docs makes things more discoverable? Hopefully? Better than scattering docs across the repo?

/shrug

Reply:
herm well I  guess I did put the README here, though there is another README in docs that links to it. idk :p